docs/ops/sequence/AUGRUSequence_dev.md

```diff
@@ -0,0 +1,168 @@
+# AUGRUSequence  {#openvino_docs_ops_sequence_AUGRUSequence_dev}
+
+**Versioned name**: *AUGRUSequence-dev*
+
+**Category**: *Sequence processing*
+
+**Short description**: *AUGRUSequence* operation represents a series of AUGRU cells (GRU with attentional update gate).
+
+**Detailed description**: The main difference between *AUGRUSequence* and [GRUSequence](./GRUSequence_5.md) is the additional attention score input `A`, which is a multiplier for the update gate.
+The AUGRU formula is based on the [paper arXiv:1809.03672](https://arxiv.org/abs/1809.03672).
+
+```
+AUGRU formula:
+  *  - matrix multiplication
+ (.) - Hadamard product (element-wise)
+
+ f, g - activation functions
+ z - update gate, r - reset gate, h - hidden gate
+ a - attention score
+
+  rt = f(Xt*(Wr^T) + Ht-1*(Rr^T) + Wbr + Rbr)
+  zt = f(Xt*(Wz^T) + Ht-1*(Rz^T) + Wbz + Rbz)
+  ht = g(Xt*(Wh^T) + (rt (.) Ht-1)*(Rh^T) + Rbh + Wbh)  # 'linear_before_reset' is False
+
+  zt' = (1 - at) (.) zt  # multiplication by attention score
+
+  Ht = (1 - zt') (.) ht + zt' (.) Ht-1
+```
+
+Activation functions for gates: *sigmoid* for f, *tanh* for g.
+Only `forward` direction is supported, so `num_directions` is always equal to `1`.
+
+**Attributes**
+
+* *hidden_size*
+
+  * **Description**: *hidden_size* specifies hidden state size.
+  * **Range of values**: a positive integer
+  * **Type**: `int`
+  * **Required**: *yes*
+
+The rest of the attributes are supported only with default values:
```

Comment:
The same comment

Reply:
Ok, I've removed this note. Just thought it would be useful as it's going to be more internal spec.
Also I removed the "Supported values: " section and updated the "Range of values: ".